lib/transform.js

```diff
@@ -3,7 +3,8 @@
 var _             = require('underscore')
   , anchor        = require('anchor-markdown-header')
   , updateSection = require('update-section')
-  , getHtmlHeaders = require('./get-html-headers');
+  , getHtmlHeaders = require('./get-html-headers')
+  , md            = require('markdown-to-ast');
```

Comment:
No, I was mostly just curious. `markdown-to-ast` looks fine. I only really asked because I know that `marked` boasts about how fast it is, but It looks like you have to do a bit of work to expose the underlying AST, and if it still doesn't give line numbers like you mention, then it's probably not worth it.